src/Vts.Scripting/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Vts.Scripting.Test")]
+namespace Vts.Scripting;
+
+/// <summary>
+/// Top program used to execute the desired demo script(s)
+/// </summary>
+public class Program
+{
```

Comment:
Have you considered having the minimal program.cs file and moving the run demos into their own class?

It is so much easier to read for a newbie than having the Main.

Reply:
I can do that!